ux.symfony.com/assets/styles/_live_demos.scss

```diff
@@ -1,3 +1,16 @@
 .form-no-mb .mb-3 {
     margin-bottom: 0 !important;
 }
+
+.form-control {
+    color: $n-900;
+}
+
+.form-control::-moz-placeholder {
+    color: $n-300;
+    font-weight: lighter;
+}
```

Comment:
It looks like this isn't needed - Firefox supports `::placeholder` since a round 2017 (per https://caniuse.com/?search=placeholder).